snap/snapcraft.yaml

```diff
@@ -0,0 +1,25 @@
+name: exa
+version: '0.7.0'
+summary: Replacement for 'ls' written in Rust
+description: |
+  It uses colours for information by default, helping you distinguish between
+  many types of files, such as whether you are the owner, or in the owning
+  group. It also has extra features not present in the original ls, such as
+  viewing the Git status for a directory, or recursing into directories with a
+  tree view. exa is written in Rust, so it’s small, fast, and portable.
+
+grade: stable
+confinement: strict
```

Comment:
How are you thinking you want to use `exa` against things outside the snap? If you are concerned with running `exa` in general, all seems well on my end. I am happy to test functionality for you with the snap installed to see what the limitation are.